cmd/process-agent/collector.go

```diff
@@ -23,6 +23,9 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+// rtProcessQueueSize is hard-coded to limit the rt process message queue to have only small number of messages
+const rtProcessQueueSize = 5
```

Comment:
We should probably make it configurable?

Reply:
I'm curious, what would be the behavior if we set it to 0? I think having a small size will mean that it takes up less space, meaning the process payload queue can be bigger, but at the end of the day, we probably don't even care about retrying RT payloads at all. What do you think?

Reply:
First off, I don't think 0 would work as it would not be able to hold anything in the queue. Also, from my observation, occasionally for whatever reason, the queue would remove more than one item from it on full queue, so we probably need more than 1 as the size. I do think having a small size makes more sense as it guarantees non-empty payloads are present so we don't skip data, while not sending too much data to intake on recovery.

Reply:
I'm not totally sure what's the point of making it configurable though, but I could add one just for the unforeseeable circumstance.

Reply:
Added!